server/pages/index.js

```diff
@@ -342,13 +342,77 @@ function setUpLoggedInRoute( req, res, next ) {
 	}
 }
 
-function setUpRoute( req, res, next ) {
-	if ( req.cookies.wordpress_logged_in ) {
-		// the user is probably logged in
-		setUpLoggedInRoute( req, res, next );
-	} else {
-		setUpLoggedOutRoute( req, res, next );
+/**
+ * Sets up a Content Security Policy header
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+ * @param {Object} req Express request object
+ * @param {Object} res Express response object
+ * @param {Function} next a callback to call when done
+ */
+function setUpCSP( req, res, next ) {
+	if ( ! req.originalUrl.match( /^\/log-in\// ) ) {
+		next();
+		return;
 	}
+
+	const inlineScripts = [
+		'sha256-3yiQswl88knA3EhjrG5tj5gmV6EUdLYFvn2dygc0xUQ=',
+		'sha256-ZKTuGaoyrLu2lwYpcyzib+xE4/2mCN8PKv31uXS3Eg4=',
+	];
```

Comment:
not sure where these are coming from, but it would be very very good to have a comment explaining from where and how to update them.

I was going to try and provide an example comment but actually I have no idea how we created them.

Reply:
You're right, I'll add an explanation.